erica/erica_legacy/request_processing/erica_input/v2/grundsteuer_input_grundstueck.py

```diff
@@ -0,0 +1,88 @@
+from enum import Enum
+from typing import Optional, List, Literal
+
+from pydantic import root_validator, validator, constr
+
+from erica.erica_legacy.request_processing.erica_input.v2.camel_case_input import CamelCaseInput
+
+
+class Grundstuecksart(str, Enum):
+    einfamilienhaus = "einfamilienhaus"
+    zweifamilienhaus = "zweifamilienhaus"
+    wohnungseigentum = "wohnungseigentum"
+    baureif = "baureif"
+    abweichende_entwicklung = "abweichendeEntwicklung"
+
+    def is_bebaut(self):
+        if self in [Grundstuecksart.einfamilienhaus, Grundstuecksart.zweifamilienhaus, Grundstuecksart.wohnungseigentum]:
+            return True
+        else:
+            return False
```

Comment:
I think we can simplify this to:
```suggestion
    def is_bebaut(self):
        return self in [Grundstuecksart.einfamilienhaus, Grundstuecksart.zweifamilienhaus, Grundstuecksart.wohnungseigentum]
```
And veery nice idea btw!

Reply:
Done